internal/wasm/sdk/test/e2e/external_test.go

```diff
@@ -204,7 +198,26 @@ func assertResultSet(t *testing.T, want []map[string]interface{}, sortBindings b
 	if !a.Equal(b) {
 		t.Fatalf("expected %v but got %v", a, b)
 	}
+}
 
+func assertErrorCode(t *testing.T, expected string, actual error) {
+	t.Helper()
+	switch expected {
+	case "eval_conflict_error":
+		exps := []string{"var assignment conflict", "object insert conflict"}
+		found := false
+		for _, exp := range exps {
+			if strings.Contains(actual.Error(), exp) {
+				found = true
+				break
+			}
+		}
+		if !found {
+			t.Errorf("expected %q to contain one of %v", actual, exps)
+		}
```

Comment:
Nice! We should never hit the default branch until we introduce some new way for policies to error.